rgd/settings.py

```diff
@@ -104,6 +104,25 @@ class ProductionConfiguration(RgdMixin, ProductionBaseConfiguration):
     pass
 
 
+class AWSProductionConfiguration(RgdMixin, ProductionBaseConfiguration):
+    AWS_S3_ACCESS_KEY_ID = None
+    AWS_S3_SECRET_ACCESS_KEY = None
+    AWS_S3_REGION_NAME = None
+    SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'http')
+
+    @staticmethod
+    def before_binding(configuration: Type[ComposedConfiguration]):
+        # Create a new file handler
+        configuration.LOGGING['handlers']['file'] = {
+            'level': 'NOTSET',
+            'class': 'logging.FileHandler',
+            'filename': '/web-apps/logs/rgd.log',
```

Comment:
Should we use an environment variable to set this path?

Reply:
https://12factor.net/logs
A more standard practice for 12-factor apps, even if they're running on EC2, is to log all output to stdout. The init system (supervisor, systemd, etc.) can then be responsible for routing output to log files (which are often stored under `/var/log` or `journalctl` with systemd) and handle tasks like log rotation.

An important reason you wouldn't want a webserver to write to a log file itself is concurrency. If you have multiple webserver processes (which is how a pre-fork WSGI server like Gunicorn scales), each will independently open and start writing to the same file simultaneously.

Reply:
Thanks for the tip!